src/ui/ui_mappings.js

```diff
@@ -19,7 +19,12 @@ class MappingsCollection {
   }
 
   getCombined = () => {
-    return this._currentMappings;
+    return {
+      doc: {
+        dynamic: false,
```

Comment:
any reason for the switch to false?  i'd think the error messages would be more noticeable than a silent failure